### ISSUE TITLE
What happens if two QGIS layers have the same name?

### ISSUE THREAD

**Issue by tomchadwin** [Geolicious/qgis2leaf#202]
I think this leads to problems if two exported layers have the same name - can anyone else confirm?

**Comment by riccardoklinger** [comment]
Yap. this whole thing is name based. should be based on layer.id() but
hasn't thought about it when creating. Yet the purpose is to export a
"finished" QGIS project which "should" not have identical layer names..

_______________________________

Geolicious GbR
Agentur für Geo-Kommunikation

Riccardo Klinger
Geschäftsführer

mobil: 0176 63437298
e-mail: riccardo.klinger@geolicious.de
web: www.geolicious.de

Geolicious GbR
c/o Daniel Kerkow
Binzstraße 48
13189 Berlin

Gesellschafter:
Riccardo Klinger
Daniel Kerkow
Jakob Hafner

Sitz der Gesellschaft: Berlin
Ust-ID: DE289107523

**Comment by tomchadwin** [comment]
Just investigating if there is a temporary workaround by giving layers aliases in QGIS.

**Comment by tomchadwin** [comment]
Yes, if you rename the layers in QGIS, all is well, of course. Something we need to look at, though.
